torch/testing/_internal/common_methods_invocations.py

```diff
@@ -751,6 +751,25 @@ def to_float(start, end, step):
     yield SampleInput(1, args=(3, 1))
 
 
+def sample_inputs_uniform(op, device, dtype, requires_grad, **kwargs):
+
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=False)
+    samples = (
+        (make_arg((S, S), dtype=dtype, device=device), 0, 1),
```

Comment:
The `make_arg` partial already specifies `dtype` and `device` so they don't need to be reiterated here.

Reply:
We should also add a few more samples, perhaps 1-d shape, and different bounds.

Reply:
fixed